graphql/resolvers.go

```diff
@@ -965,6 +965,44 @@ func (r *queryResolver) ProjectSettings(ctx context.Context, identifier string)
 	return res, nil
 }
 
+func (r *mutationResolver) CreateProject(ctx context.Context, project restModel.APIProjectRef) (*restModel.APIProjectRef, error) {
+	projectRef, err := model.FindOneProjectRef(*project.Identifier)
+	if err != nil && err.(gimlet.ErrorResponse).StatusCode != http.StatusNotFound {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error looking in project collection: %s", err.Error()))
+	}
+	if projectRef != nil {
+		return nil, InputValidationError.Send(ctx, fmt.Sprintf("cannot create project with identifier '%s'", err.Error()))
+	}
+
+	i, err := project.ToService()
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, errors.Wrapf(err, "API error converting from model.APIProjectRef to model.ProjectRef").Error())
+	}
+	dbProjectRef, ok := i.(*model.ProjectRef)
+	if !ok {
+		return nil, InternalServerError.Send(ctx, errors.Wrapf(err, "Unexpected type %T for model.ProjectRef", i).Error())
+	}
+
+	u := gimlet.GetUser(ctx).(*user.DBUser)
+	if err = r.sc.CreateProject(dbProjectRef, u); err != nil {
+		return nil, InternalServerError.Send(ctx, errors.Wrapf(err, "Database error for insert() distro with distro id '%s'", *project.Identifier).Error())
```

Comment:
copy pasta error

Reply:
haha I think this is a copy pasta error, [of a previous copy pasta error](https://github.com/malikchaya2/evergreen/blob/EVG-15039/rest/route/project.go#L590).